src/Model/ModelUtilities/BoundaryPackage.f90

```diff
@@ -1669,7 +1659,8 @@ subroutine bnd_bd_obs(this)
     do i=1,this%obs%npakobs
       obsrv => this%obs%pakobs(i)%obsrv
       if (obsrv%BndFound) then
-        do n=1,size(obsrv%indxbnds)
+        !do n = 1, size(obsrv%indxbnds)
+        do n = 1, obsrv%indxbnds_count
```

Comment:
Looks good how you introduced a member variable instead of using Count.  Makes sense to me.

Reply:
ok